src/main/scala/cromwell/binding/values/WdlValueJsonFormatter.scala

```diff
@@ -11,7 +11,11 @@ object WdlValueJsonFormatter extends DefaultJsonProtocol {
       case f: WdlFloat => JsNumber(f.value)
       case b: WdlBoolean => JsBoolean(b.value)
       case f: WdlFile => JsString(f.value)
-      case o: WdlObject => JsObject()
+      case o: WdlObject => new JsObject(
```

Comment:
About this, I nested the actual json representation into an 
```json
{ "object": {
  "foo" : "bar",
    ...
  }
}
```
to make the difference with a map. Not sure this is a the best approach. Any idea ?